core/templates/dev/head/pages/exploration_player/correctness_footer_directive.html

```diff
@@ -25,7 +25,41 @@
   }
 
   correctness-footer .inner-content {
-    padding-left:8px;
+    padding-left: 8px;
+  }
+
+  @-webkit-keyframes correct_feedback_fade_animation {
```

Comment:
will it be compatible with other browsers? I think we should add at least @-moz-keyframes
(see [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/css/oppia.css#L40) we already have fade in animation maybe you can use it)

Reply:
I tried Firefox 64-bit v57.0.4 in Ubuntu 16.04 an it worked, should I add it anyways?

Reply:
If we have existing code that achieves the same thing, I would rather go with that. 

Though in terms of compatibility, I'm more concerned with IE and MS Edge (the IE successor). I tested it on FF with the same version that @aks681 has and it works as expected on my side.

Reply:
Done, I added support MS browsers and changed to using the already existing fade-in class.
@kevinlee12, for a second opinion, as per @DubeySandeep suggestion, should I do slide-in?

Reply:
not a fan of a slide-in because it's a lot of movement in a situation where there's already a lot of movement